data/blog/electron-6-0.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Electron 6.0.0
+author: 
+- sofianguy
+date: '2019-07-30'
+---
+
+The Electron team is excited to announce the release of Electron 6.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). The release is packed with upgrades, fixes, and new features. We can't wait to see what you build with them! Continue reading for details about this release, and please share any feedback you have!
+
+---
+
+## What's New
+
+This is the first Electron stable release to ever release on the *same day* as [Chrome stable](https://www.chromestatus.com/features/schedule) 🎉
+
+Much of Electron's functionality is provided by the core components of Chromium, Node.js, and V8. Electron keeps up-to-date with these projects to provide our users with new JavaScript features, performance improvements, and security fixes. Each of these packages has a major version bump in Electron 6:
+
+- Chromium `76.0.3809.77`
+  - [New in 74](https://developers.google.com/web/updates/2019/04/nic74)
+  - [New in 75](https://developers.google.com/web/updates/2019/06/nic75)
+  - [New in 76](https://developers.google.com/web/updates/tags/chrome76)
+- Node.js `12.4.0`
+  - [Node 12.4.0 blog post](https://nodejs.org/en/blog/release/v12.4.0/)
+- V8 `7.6.303.22`
+    - [V8 7.6 blog post](https://v8.dev/blog/v8-release-76)
+
+Electron 6 also includes improvements to Electron-specific APIs. Highlights of the major changes is below; for the full list of changes, check out the [Electron v6.0.0 release notes](https://github.com/electron/electron/releases/tag/v6.0.0).
+
+### Promisification
+
+Electron 6 continues [Promisification initiative](https://github.com/electron/electron/blob/5-0-x/docs/api/promisification.md) initiative to convert Electron's callback-based API to use Promises. These APIs were converted for Electron 6:
```

Comment:
continues and completes?

cc @codebytere can you confirm that all the promisification work is done and in the `6-0-x` branch

Reply:
Resolved via Slack. @codebytere says promisification are not done yet in v6.